src/components/wallet/PageTransactions.vue

```diff
@@ -195,6 +195,10 @@ export default {
             ...this.loadedTransactions,
             ...result.transactionsV2
           ]
+          // sorting transactions
+          this.loadedTransactions = this.loadedTransactions.sort(
+            (a, b) => b.height - a.height
```

Comment:
This could lead to shuffling of messages in one tx as we have no way to sort them. Added an issue to lunie-api.

Reply:
Why? They are sorted in a component

Reply:
there can be multiple transactions per height even per hash right? How do you sort the transactions then?

Reply:
But they are already sorting somewhere in transaction list component. Even after this sort method.
I have added it just for one purpose. We have two possible update calls: to bottom when older transactions are loaded and to top — when new ones are loaded from subscribeToMore.
And in both cases we make result = [...result, ...newTransactions] so the new ones are on the bottom of the list.
But transactions are shown with limitation, so only the first ones are sending to transactions list. 
It's ok when we are adding older records, but wrong when we are added new records.
That's why i have to sort them.

Reply:
This is a correct implementation I think. That's why I did approve the PR already. But it surfaced that there is not enough sorting in the data model. See this issue https://github.com/luniehq/lunie-api/issues/477